keps/sig-scheduling/4664-guarantee-pdb-when-preemption-happens/README.md

```diff
@@ -369,15 +373,11 @@ when selecting victims.
 - if the priority of the preemptor is greater than or equal to the value of `AllowDisruptionByPriorityGreaterThanOrEqual` in victim pod, 
   the implementation will remain consistent with the existing behavior, meaning that the scheduler will try to select victims whose PDBs 
   are not violated by preemption, but if no such victims are found, preemption will still happen and lower priority pods will be preempted 
-  via the `/evictions` endpoint despite their PDBs being violated. If the `/eviction` endpoint returns a response `429 Too Many Requests`
-  and the scheduler will fallback to deletion as an alternative.
+  using the client-go's delete method.
 - if the priority of the preemptor is less than the value of `AllowDisruptionByPriorityGreaterThanOrEqual` in victim pod, 
   the scheduler will check if the victim' PDBs will be violated when selecting victims
   - if violate the victims' PDBs, this victim will not be selected as candidates.
-  - if not violate the victims' PDBs, scheduler will preempt this pod via the `/evictions` endpoint. If it responds `200 OK`, 
-    it means the eviction is allowed, and the victim is deleted, similar to sending a DELETE request to the Pod URL. If it responses
-    `429 Too Many Requests`, the scheduler will output an error log and choose another victim among the candidate victims to preempt 
-    until it succeeds or there are no more candidates.
+  - if not violate the victims' PDBs, scheduler will preempt this pod using the client-go's delete method.
```

Comment:
Using delete introduces the conflicts that we are trying to avoid. Removing the evictions would defeat the purpose of this feature.